cli/objects/jira_base.py

```diff
@@ -308,3 +308,11 @@ def _get_security_level_id(
             f"Security level {security_level} not found in {project_key}, no security level will be applied.",
         )
         return None
+
+    def get_issue_by_id_or_key(self, issue_id_or_key: str) -> Issue:
```

Comment:
What is the meaning of issue by key?
I think docstring is missing here.

Reply:
This is in reference to the Jira API's Issue objects which have both a "key" field and an "id" field. The endpoint used by this command works with either, and both fields are used in our code.